src/Modules/StatCan.OrchardCore.Ethos/Recipes/Persona.recipe.json

```diff
@@ -0,0 +1,95 @@
+{
+    "name": "Persona",
+    "displayName": "Persona",
+    "description": "Creates a Persona tool.",
+    "author": "StatCan Digital Innovation",
+    "website": "digital.statcan.gc.ca",
+    "version": "1.0.0",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [], 
+    "steps": [
+      {
+        "name": "ContentDefinition",
+        "ContentTypes": [
+          {
+            "Name": "Persona",
```

Comment:
I don't think this is required since you are creating the types with the code Migration